packages/patternfly-4/react-docs/README.md

```diff
@@ -26,7 +26,7 @@ import localDep from './examples/exampleDep.js'
 
 Remember that you are unable to use experimental language features like [class properties](https://babeljs.io/docs/en/babel-plugin-proposal-class-properties) as [Buble](https://github.com/bublejs/buble) does not support them.
 
-## About Gatsby
+## About gatsby
```

Comment:
I'd argue that Gatsby can stay capitalized. Otherwise it's like referring to Red Hat as redhat.